README.md

```diff
@@ -184,30 +210,24 @@ ansible-playbook drop-databases.yml                                \
     --user       $user
 ```
 
-### Migrate or reset the database
+### Deploy or delete the secrets
+This is a step that allows to modify kubernetes secrets.
+In normal scenario, we don't needed to use them, only when secrets are updated or clear all component on the cluster.
```

Comment:
Please put this section after `Uploading secrets, building containers and cluster configuration`.

And rename `Uploading secrets, building containers and cluster configuration` to `Building containers and cluster configuration`.

Reply:
Done.